libcrux-ml-dsa/Cargo.toml

```diff
@@ -20,7 +20,8 @@ libcrux-sha3 = { version = "0.0.2-beta.3", path = "../libcrux-sha3" }
 libcrux-intrinsics = { version = "0.0.2-beta.3", path = "../libcrux-intrinsics" }
 libcrux-platform = { version = "0.0.2-beta.3", path = "../sys/platform" }
 libcrux-macros = { version = "0.0.2-beta.3", path = "../macros" }
-hax-lib = { version = "0.1.0", git = "https://github.com/hacspec/hax/"}
+hax-lib = { version = "0.1.0", git = "https://github.com/hacspec/hax/", branch = "replace-body-temp"}
+minicore = { path = "../fstar-helpers/minicore" }
```

Comment:
So this requires releasing the minicore and actually making it releasable.

Reply:
Ideally, I'd like to kill that Rust dependency. I have a plan for that but I need to experiment a little: that will require some more rewriting in F*.

Reply:
So: this dependency is not needed in libcrux-intrsinsics, but I still need those in ml-dsa, but only on specs

Reply:
Well, it's a dependency, so it needs to be released. `path` doesn't work for that.